packaging/arch/PKGBUILD

```diff
@@ -0,0 +1,15 @@
+pkgname=cockpit-podman
+pkgver=VERSION
+pkgrel=1
+pkgdesc='Cockpit UI for podman containers'
+arch=('any')
+url='https://github.com/cockpit-project/cockpit-podman'
+license=(LGPL)
+depends=(cockpit podman)
```

Comment:
I assume you copied this from the official arch repo? Do you have a pointer to the cockpit one? In Debian/Ubuntu/Fedora/RHEL we split the cockpit source package into multiple binaries: cockpit-bridge (arch dependent), cockpit-ws (webserver), cockpit-system (login, shell, most important pages), and a bunch of others like cockpit-networkmanager. "cockpit" binary package is just an empty meta-package that depends/recommends the former.

Usually "extension" packages like c-podman should only depend on cockpit-bridge, not the full cockpit meta-package, to avoid dragging in the webserver on every installation (you can use it on a remote host through ssh).

No blocker for this PR of course, but worth discussing.

Reply:
I see, thanks! So only cockpit-pcp is split out, then this is correct.